cmd/nerdctl/image_convert.go

```diff
@@ -67,6 +68,16 @@ func newImageConvertCommand() *cobra.Command {
 	imageConvertCommand.Flags().Bool("zstdchunked", false, "Use zstd compression instead of gzip (a.k.a zstd:chunked). Should be used in conjunction with '--oci'")
 	// #endregion
 
+	// #region nydus flags
```

Comment:
Are all these flags production ready?
Some might need to be experimental?
https://github.com/containerd/nerdctl/blob/master/docs/experimental.md

Reply:
The option `--nydus-chunk-dict` is experimental. I can just remove it.